data_handler/handlers/loan_states/hashtack_v0/interest_rate.py

```diff
@@ -0,0 +1,81 @@
+from handler_tools.api_connector import DeRiskAPIConnector
+from handlers.loan_states.hashtack_v0.events import HashstackV0Event
+
+from decimal import Decimal
+
+
+HASHSTACK_ADDRESS = "0x01b862c518939339b950d0d21a3d4cc8ead102d6270850ac8544636e558fab68"
```

Comment:
Please add more specific name
```suggestion
HASHSTACK_INTEREST_RATE_ADDRESS = "0x01b862c518939339b950d0d21a3d4cc8ead102d6270850ac8544636e558fab68"
```